summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -114,21 +114,25 @@ gitHubFiles Settings{..} = concat
     ghActionsYml = unlines $
         [ "name: CI"
         , ""
-        , "# Trigger the workflow on push or pull request, but only for the main branch"
         , "on:"
         , "  pull_request:"
+        , "    types: [synchronize, opened, reopened]"
         , "  push:"
         , "    branches: [main]"
+        , "  schedule:"
+        , "    # additionally run once per week (At 00:00 on Sunday) to maintain cache"
+        , "    - cron: '0 0 * * 0'"
         , ""
         , "jobs:"
+
         ]
         <> memptyIfFalse settingsCabal ghActionsCabal
         <> memptyIfFalse (settingsCabal && settingsStack) [""]
         <> memptyIfFalse settingsStack ghActionsStack
 
-    ghcActionsCheckoutVersion = "@v2.3.3"
-    ghcActionsSetupHaskellVersion = "@v1"
-    ghcActionsCacheVersion = "@v2.1.2"
+    ghcActionsCheckoutVersion = "@v2"
+    ghcActionsSetupHaskellVersion = "@v1.2"
+    ghcActionsCacheVersion = "@v2"
```

Comment:
Do you think it worth moving this to defaults?

Reply:
Good idea! 👍🏻